generic/tests/cfg/timedrift.cfg

```diff
@@ -19,11 +19,13 @@
                     used_cpus = 100
                     variants:
                         - default_load:
+                            host_load_instances = 0
+                            guest_load_instances = 2
                         - full_load:
                             test_duration = 28800
                             interval_gettime = 1800
-                            pre_command = touch /tmp/host_load_timedrift; for (( I=0; I<`grep -c 'processor' /proc/cpuinfo`; I++ )); do echo $I;taskset -c $I /bin/bash -c 'while [ -e /tmp/host_load_timedrift ]; do :; done&'; done
-                            post_command = rm -f /tmp/host_load_timedrift
+                            host_load_instances =
+                            guest_load_instances =
```

Comment:
With the change of avocado-vt and kar,  no default value for them,  And no default value in timedrift.py, will meet no key error.
```
   guest_load_instances = int(params["guest_load_instances"])
   host_load_instances = int(params["host_load_instances"])
```